x/plans/keeper/migrations.go

```diff
@@ -154,3 +155,74 @@ func (m Migrator) Migrate7to8(ctx sdk.Context) error {
 
 	return nil
 }
+
+func (m Migrator) Migrate8to9(ctx sdk.Context) error {
+	adventurerLimit := uint64(1)
```

Comment:
how did you decide those numbers?

Reply:
using Sid's spreadsheet (from the subscription rates design)

Reply:
the gateway needs unlimited projects as of now.
this is the whale plan, the rest you can leave as it is